ChangeLog.txt

```diff
@@ -6,6 +6,8 @@ MAJOR:
 - Redesigned the dejure structure of Yemen and reworked Socotra
 - Reworked many colours across the map, based upon: https://forum.paradoxplaza.com/forum/index.php?threads/wtwsms-project-colouring-the-world.915304/
 - Overhauled the map of Mesopotamia and Western Persia, see on: http://forum.paradoxplaza.com/forum/index.php?threads/wtwsms-presentation-info-news-links.829142/page-4#post-21192710
+- Hypsistarians are no longer their own lonely group
```

Comment:
Well, they need some features. I still don't get the point of putting them in their own religious group. What does that accomplish except making them basically unplayable?